proto/osmosis/gamm/v1beta1/query.proto

```diff
@@ -282,7 +283,7 @@ message QuerySwapExactAmountOutRequest {
   // TODO: CHANGE THIS TO RESERVED IN A PATCH RELEASE
   string sender = 1 [ (gogoproto.moretags) = "yaml:\"sender\"" ];
   uint64 pool_id = 2 [ (gogoproto.moretags) = "yaml:\"pool_id\"" ];
-  repeated SwapAmountOutRoute routes = 3 [
+  repeated osmosis.swaprouter.v1beta1.SwapAmountOutRoute routes = 3 [
     (gogoproto.moretags) = "yaml:\"routes\"",
     (gogoproto.nullable) = false
   ];
```

Comment:
We should ask dan lynch and front-end how breaking this is.

Its not clear to me that its worth breaking this in the gamm proto, vs just doing a conversion in the proto boilerplate handling code.

Reply:
(I think it will break their compiled types, unless we can register a way for a type to get aliased or something in codegen)

Reply:
as long as the interfaces/API don't change, it doesn't matter where the types live. Telescope will handle it.

from [what I can see here](https://github.com/osmosis-labs/osmosis/blob/f594d0d6614a46f4cbf5dfb6af71ace3828b41d9/proto/osmosis/swaprouter/v1beta1/swap_route.proto#L8-L18) it looks the the interface to the type doesn't change, so will be totally ok!

Reply:
also, even w/o telescope, from what I can tell (since I believe osmosis-fe uses another tool) — nothing should break. But, we should probably all keep in sync when this update rolls out to update FE libs and repos.

Reply:
Thanks, @pyramation . I also spoke with Jon to make sure that there is no unexpected breakage for FE.

We will track this change and remind everyone to update before this is out.

This should be good to go